app/views/staff/organisations/show.html.haml

```diff
@@ -1,16 +1,19 @@
 =content_for :page_title_prefix, t("page_title.organisation.show", name: @organisation_presenter.name)
 
-= link_to t("generic.link.back"), organisations_path, class: "govuk-back-link"
+= organisation_page_back_link(current_user, params)
+
 %main.govuk-main-wrapper#main-content{ role: "main" }
   .govuk-grid-row
-    .govuk-grid-column-two-thirds
+    .govuk-grid-column-full
       %h1.govuk-heading-xl
         = t("page_title.organisation.show", name: @organisation_presenter.name)
 
   .govuk-grid-row
     .govuk-grid-column-two-thirds
       %h2.govuk-heading-m Details
-      = link_to t("page_content.organisation.button.edit"), edit_organisation_path(@organisation_presenter), class: "govuk-button"
+
+      - if policy(Organisation).edit?
```

Comment:
I notice we used `edit?` here, I was expecting `update?` which threw me for a bit, is there a reason to use edit instaed?

Reply:
No, just an error I think!